ACE/ace/CDR_Base.inl

```diff
@@ -145,7 +145,11 @@ ACE_CDR::swap_8 (const char* orig, char* target)
     && !defined(ACE_LACKS_INLINE_ASSEMBLY)
   ACE_REGISTER unsigned long x =
     * reinterpret_cast<const unsigned long*> (orig);
+#if defined (__MINGW64__)
```

Comment:
You are right. I made patches for Ubuntu 12.04/14.04 x86_64-w64-mingw32-g++. 

I don't know how to report them to Debian/Ubuntu upstream. So I made PPA for them:

https://launchpad.net/~likemartinma/+archive/ubuntu/devel

They can make ACE cross compiling successfully. 

I used them into our company product, an enterprise level heterogeneous backup and recovery suite, just like Veritas NetBackup. It works very well so far.

BTW, thanks ACE and you guys. It makes our product catching up with Veritas NetBackup in Chinese market.